imageroot/events/certificate-updated/00validate_event

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see COPYING.
+#
+
+import json
+import sys
+import agent
+import os
+
+event = json.load(sys.stdin)
+
+if str(event['node']) != os.environ['NODE_ID']:
+    print(agent.SD_DEBUG + "Event ignored: source is another node")
+    sys.exit(2)
```

Comment:
Maybe it's better to exit with 0 code? It's not an error, but a normal condition. Also on line 36

Reply:
Event output and exit codes are ignored: nobody can check them. A non-zero exit code from an handler step has the only effect of aborting the whole event handler: following steps are not executed.

Reply:
So it's not really needed, better to merge everything inside one action.